src/test/java/me/karakelley/http/HttpServerTest.java

```diff
@@ -257,7 +274,8 @@ void testDisplaysFilesAtRoot() {
       } catch (IOException e) {
         e.printStackTrace();
       }
-      assertTrue(response.size() == 6);
+      assertTrue(response.contains("Content-Type: text/html"));
+      assertTrue(response.size() > 4);
```

Comment:
What's this magic number `4` all about? And why the change from a precise `==` to a more lenient `>`?